crates/consensus/src/consensus.rs

```diff
@@ -46,7 +46,7 @@ impl Consensus for BeaconConsensus {
         verification::validate_header_standalone(header, &self.chain_spec)?;
         verification::validate_header_regarding_parent(parent, header, &self.chain_spec)?;
 
-        if Some(header.number) < self.chain_spec.paris_status().block_number() {
+        if !self.chain_spec.fork_active(Hardfork::MergeNetsplit, header.number.into()) {
```

Comment:
We need to add a note here (or fix it here) about this not being entirely sound. If the network we are running on has not merged yet, we won't know the block, so we should go off of TTD.